tests/test_django/app/migrations/0001_initial.py

```
@@ -1,12 +1,11 @@
-# Generated by Django 5.1.2 on 2024-10-26 08:49
+# Generated by Django 5.1.4 on 2025-04-04 05:32
```

Ruff (0.8.2)

Missing docstring in public module

(D100)